source/common/config/grpc_stream.h

```diff
@@ -129,8 +129,8 @@ class GrpcStream : public Grpc::AsyncStreamCallbacks<ResponseProto>,
   GrpcStreamCallbacks<ResponseProto>* const callbacks_;
 
   // TODO(htuch): Make this configurable or some static.
-  const uint32_t RETRY_INITIAL_DELAY_MS = 500;
-  const uint32_t RETRY_MAX_DELAY_MS = 30000; // Do not cross more than 30s
+  const uint32_t retry_initial_delay_ms_ = 500;
```

Comment:
make this static for now and keep the case? this is constant right?

Reply:
done